### PR TITLE
Fix issues with incorrect offsets being returned.

### PR DESCRIPTION
**Pull request by sampritipanda**

### CODE REVIEW

File: tsearch_extras.c [hunk 1]
```diff
@@ -74,9 +77,13 @@ ts_match_locs_setup(Oid cfgId, TsMatchesData *mdata, text* in, TSQuery query)
 
 	hlparsetext(cfgId, &prs, query, VARDATA(in), VARSIZE(in) - VARHDRSZ);
 
+	headline_options = lappend(headline_options,
+											  makeDefElem(pstrdup("HighlightAll"),
+											  (Node *) makeString(pstrdup("1"))));
```
Comment (timabbott) [inline tsearch_extras.c:82]: This seems way over-indented, doesn't it?

**Comment by timabbott** [comment]
Have you tested this both with 9.3 and 9.5?  I'm wondering if we may need to include some `#ifdefs` here.

File: tsearch_extras.c (hunk 1 above)
Comment (sampritipanda) [reply tsearch_extras.c:82]: That's how the rest of the file is indented using tabs (https://github.com/sampritipanda/tsearch_extras/blob/a4277cb22a918dfc6bc2fd4009f75c2152f2ad83/tsearch_extras.c#L171)

Should I convert all of it to spaces?

**Comment by sampritipanda** [comment]
Yeah, this works fine on both versions.

File: tsearch_extras.c (hunk 1 above)
Comment (timabbott) [reply tsearch_extras.c:82]: Oh, I see.  Hmm, we probably shouldn't change it, since I bet the code has the same style rules as Postgres upstream.

**Comment by timabbott** [comment]
I'm testing this out in production now; seems to be working so far.  I'll merge and do a release once confirmed.

**Pull request merged**